src/App.jsx

```diff
@@ -17,10 +18,96 @@ function App() {
     });
   }, []);
 
+
+  function showFiles() {
+    let arrow = document.querySelectorAll(".arrow");
+    for (var i = 0; i < arrow.length; i++) {
```

Comment:
We'll change this up to do it "the React way", but this is great as a first pass.